### ISSUE TITLE
All icons should have a default 'Alt' description 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/Icons#165]
## Is your feature request related to a problem? Please describe.

For accessibility reasons, SVGs should be updated to support a proper default position. 

## Describe the solution you'd like

For properly accessible SVG files, we SHOULD have the following. 

```
<svg> + role="img" + <title> + <desc> + aria-labelledby="[ID]"
```

With research and testing, it is understood that `title` is the content that is shown via tool-tip in the UI and read aloud by screen readers. 

The `desc` node is recommended, but I found little evidence that this content is ever read back to users. 

We cannot pre-define the `aria-labelledby` attribute, but we can bring back [this content](https://github.com/AlaskaAirlines/Icons/blob/584c3ed5509883cdd31f71daa792cbeab6fa5c74/README.md#altering-the-svg-output) that illustrates how users can edit the output of the SVG on demand. 

It is my recommendation we update all the icons to have a descriptive title and the use of `desc` is redundant and should be removed. 

## Exit criteria 

This issue will be considered complete once it is verified that all icons have this pattern.
